src/i18n/en.js

```diff
@@ -215,6 +215,31 @@ const messages_en = {
       transfer: 'Bank transfer',
       valid_thru: 'valid thru',
     },
+    purchase_summary: {
+      buy_button: 'BUY',
+      discount_for_payment_paid: 'Positive balance:',
+      discount_for_prepayment: 'Discount for prepayment:',
+      edit_add_recipients_button: 'Edit or add recipients',
+      error_message: 'Your payment couldn’t be processed. Choose a different payment method or try it later.',
+      explanatory_legend: 'The renewal is automatic and you can cancel it whenever you want. The price of the Plan does not include taxes.',
+      header: 'Purchase summary',
+      month_with_plural: `{monthsCount, plural, one {# month}other {# months} }`,
+      months_to_pay: 'Months to pay:',
+      plan_standard_title: 'STANDARD plan',
+      plan_type_monthly_deliveries: 'by Emails',
+      plan_type_monthly_deliveries_label: 'Emails',
+      plan_type_prepaid: 'by Credits',
+      plan_type_subscribers: 'by Contacts',
+      plan_type_subscribers_label: 'Contacts:',
+      promocode_label: 'Promo code',
+      promocode_legend: 'Enter a promo code',
```

Comment:
Wouldn`t this be like this?
```suggestion
      promocode_legend: 'Enter a Promo Code',
```
@jmonacchi what do you think?

Reply:
@cbernat is there any technical reason to use Promo Code with capital letters? If not, promo code sounds more natural for me.

Reply:
In old version of upgrade it is with capitals, but perhaps as you say, it is not a Doppler entity, like Lists, Segments and so on.